PatientRegistrationApp/package.json

```diff
@@ -37,7 +40,18 @@
     "parserOptions": {
       "parser": "babel-eslint"
     },
-    "rules": {}
+    "rules": {},
+    "overrides": [
+      {
+        "files": [
+          "**/__tests__/*.{j,t}s?(x)",
+          "**/tests/unit/**/*.spec.{j,t}s?(x)"
```

Comment:
This is the default filename suffix for jest. Basically any unit test being run has to have the .spec to be recognized when test are run. For example HelloWorld.spec.js tests HelloWorld.vue. This can be changed btw.

Reply:
Ok - could we change it to be `"**/tests/unit//**/*.test.{j,t}s?(x)"` so it reads HelloWorld.test.js instead?

Reply:
Yes. I should be able to manage that.